lib/adal.js

```diff
@@ -1187,7 +1141,7 @@ AuthenticationContext.prototype._cloneConfig = function (obj) {
     return copy;
 };
 
-AuthenticationContext.prototype._addClientId = function () {
+AuthenticationContext.prototype._addXClientParameters = function () {
```

Comment:
we are adding x-client-SKU and x-client-ver parameters, thats why named it addXClientParameters.